compiler/src/iree/compiler/Codegen/Common/GPUPipelining.cpp

```diff
@@ -18,6 +19,7 @@
 #include "mlir/Dialect/Vector/IR/VectorOps.h"
 #include "mlir/Interfaces/SideEffectInterfaces.h"
 #include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+#define DEBUG_TYPE "iree-gpu-nvidia-ampere-mainloop-schedule"
```

Comment:
Replace with:
```
#define DEBUG_TYPE "iree-gpu-pipelining"
````
as this is common to the whole file

Reply:
Done